data/remnant missions.txt

```diff
@@ -56,6 +56,8 @@ mission "First Contact: Remnant"
 				decline
 			
 	on enter
+		log "People" "Chilia" `Met Chilia, a Remnant Prefect, who explained why they want everyone to take a blood test.`
```

Comment:
Logs for individual people aren't suppose to be written in the same way as normal logs. This sounds like a normal log to me, where as a `People` log for Chilia should be a description of who Chilia is, not something small that Chilia did.
If you're to rewrite this as a proper log *for* Chilia, add some information that isn't given elsewhere.
You could also just make this a normal log, but I'd still rewrite it to be something more like `Met Chilia, a Remnant Prefect, who explained that they blood test all foreign visitors because...`

Reply:
Thanks for the feedback. I've rewritten it. How does this look for a person log:
`Met Chilia, who is apparently a Remnant Prefect. He looks quite young to be holding a position of authority, but the Remnant I have seen seem to respect him.`